src/WebJobs.Extensions.DurableTask/DurableHttpResponse.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Listener;
+using Microsoft.Extensions.Primitives;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// Response received from the HTTP request made by the Durable Function.
+    /// </summary>
+    // [JsonConverter(typeof(DurableHttpResponseJsonConverter))]
+    public class DurableHttpResponse
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DurableHttpResponse"/> class.
+        /// </summary>
+        /// <param name="statusCode">HTTP Status code returned from the HTTP call.</param>
+        [JsonConstructor]
+        public DurableHttpResponse(HttpStatusCode statusCode)
+        {
+            this.StatusCode = statusCode;
+            this.Headers = new Dictionary<string, StringValues>(StringComparer.OrdinalIgnoreCase);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DurableHttpResponse"/> class.
+        /// </summary>
+        /// <param name="jObject">JObject containing information from HttpResponseMessage.</param>
+        public DurableHttpResponse(JObject jObject)
+        {
+            int codeInt = int.Parse(jObject["StatusCode"].Value<string>());
+            HttpStatusCode statusCode = (HttpStatusCode)codeInt;
+
+            Dictionary<string, StringValues> headerDictStringValues = new Dictionary<string, StringValues>();
```

Comment:
Need to use the `StringComparer.OrdinalIgnoreCase` parameter on this dictionary constructor